app/models/currency.rb

```diff
@@ -16,12 +16,9 @@ def division
     Division.root # for permissions purposes, assume currency model belongs to root division
   end
 
-  private
-
   def plural_name
-    # This should obviously be refactored someday. Ideally the currency's name field would
+    # TODO: This should obviously be refactored someday. Ideally the currency's name field would
     # be converted to be translatable. Currently the name is stored only in English.
-    # As of 9/2016 this is only used in the public front end (which is used in Spanish as well!).
     "#{name}s"
```

Comment:
Yes, this indeed needs to be refactored in the future. I'll make sure this is in the backlog.